prime-router/src/test/kotlin/metadata/MapperTests.kt

```diff
@@ -80,6 +82,178 @@ class MapperTests {
         assertThat(mapper.apply(lookupElement, args, elementAndValues).value).isEqualTo("y")
     }
 
+    private fun assertEq(
+        mapper: Mapper,
+        elm: Element,
+        args: List<String>,
+        EAVs: List<ElementAndValue>,
+        compVal: String
+    ) {
+        assertThat(
+            mapper.apply(elm, args, EAVs).value
+        ).isEqualTo(compVal)
+    }
+
+    @Test
+    fun `test IfThenElseMapper`() {
+        val mapper = IfThenElseMapper()
+        val element = Element("test")
+        var args = mutableListOf<String>()
+        while (args.count() < 11) { // test for zero to ten args passed - each should fail (skip testing 5)
```

Comment:
Suggestion: move the comment to the line above like your other comments, so it is read first and it is consistent with your other comments

Reply:
done